lib/pow/store/backend/mnesia_cache.ex

```diff
@@ -214,32 +214,24 @@ defmodule Pow.Store.Backend.MnesiaCache do
     |> Enum.each(&:rpc.call(&1, GenServer, :cast, [__MODULE__, {:refresh_invalidators, config}]))
   end
 
-  defp clear_invalidator(key, invalidators) do
-    case Map.get(invalidators, key) do
-      nil         -> nil
-      invalidator -> Process.cancel_timer(invalidator)
-    end
-
+  defp clear_invalidator(key, invalidators) when :erlang.is_map_key(key, invalidators) do
```

Comment:
This would require `Erlang 21`

Reply:
Can you revert this change? For now I think we would like to keep support for OTP 20.

Reply:
Done.